middleware/middleware-1.3.23.md

```diff
@@ -0,0 +1,68 @@
+
+## Feature: SwissbitCloud support in _Bring your own Data Center_
+Our _Bring your own Data Center_ solution now supports _SwissbitCloud_ and _DeutscheFiskal_ TSEs using self hosted FCCs. To make use of this, the Fiskal Cloud Connector should be installed on a machine or container in the same network, and the SCU configured to use it via the `FccUri` configuration parameter (more details in our [docs](https://docs.fiskaltrust.cloud/docs/poscreators/middleware-doc/germany/scu/swissbit-cloud)).
+
+## Bug fix: Bring your own Data Center didn't work with custom Redis instance
+We've fixed an issue where the connection string to custom Redis instances would be assembled wrongly.
+
+## Bug fixes: Small DSFinV-K improvements
+We've fixed three issues in our DSFinV-K exports that could occur in some edge cases: 
+- `UMS_BRUTTO` was calculated wrongly when discounts were sent in a specific format
+- `REF_BON_ID` was incorrectly set to the value of `cbReceiptReference` instead of `ftReceiptIdentification` (only applied to local exports, Portal exports were not affected)
+- Some timestamps in _transactions.csv_ were not written in UTC, but in the local timezone (only applied to local exports, Portal exports were not affected)
+
+## Bug fixes: FiskalyCertified SCU returned an empty certification ID
+We've fixed an issue where the new _FiskalyCertified_ SCU wrongly returned an empty certification ID instead of the correct value `BSI-K-TR-0403-2021`.
+
+## Documentation: Process documentation for DSFinV-K exports
+We have created documentation templates for the mandatory process documentation (_Verfahrensdokumentation_), in which we describe how the Middleware populates specific fields in the DSFinV-K export, and how exports can be downloaded from the Portal if the users bought a PosArchive package. This template can be added to the PosCreator's or PosDealer's process documentation and might be useful in case questions occur during audits.
+
+It's available in the [German Portal's download section](https://portal.fiskaltrust.de/AccountProfile/Download).
+
+## Affected packages
+Packages not listed here were not updated, as we decided to not increase the version of unchanged packages. All packages with versions greater or equal to 1.3.1 are compatible with each other (it is e.g. possible to use _fiskaltrust.Middleware.SCU.Swissbit.1.3.1_ with the new queue packages).
+
+- _fiskaltrust.Middleware.Queue.EF v1.3.23_
+- _fiskaltrust.Middleware.Queue.MySQL v1.3.23_
+- _fiskaltrust.Middleware.Queue.SQLite v1.3.23_
+- _fiskaltrust.Middleware.SCU.DE.FiskalyCertified v1.3.23_
```

Comment:
Do we want to highlight that this is the first release that is not a release candidate?

Reply:
Good point, I've added a sentence to the intro 🙂